src/main/java/org/nuxeo/tools/gatling/report/ParserFactory.java

```diff
@@ -35,7 +35,10 @@ protected static SimulationParser getVersionSpecificParser(File file, Float apde
         // System.out.println(header.size() + " " + header);
         if (header.size() == 6) {
             String version = header.get(5);
-            if (version.matches("3\\.[2-9].*")) {
+            if (version.matches("3\\.[5-9].*")) {
+                return new SimulationParserV35(file, apdexT);
+            }
+            if (version.matches("3\\.[2-4].*")) {
```

Comment:
Hi @thyzzv , I'm also looking to get support but for gatling 3.4 and the changes to the simulation log are from 3.4 onwards. Could you change this to have 3.4+ in one case? Thanks!

Reply:
Actually I opened a separate PR for 3.4 as there seem to be other format changes between 3.4 and 3.5. Please ignore the prior comment.